tools/distribution/verify.py

```diff
@@ -0,0 +1,203 @@
+import argparse
```

Comment:
Generally prefer file-level comments on what is going on in here, how this relates to other files, CLI usage examples.

Reply:
yep, good catch, i will add